src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
I don't think the naming concern is important enough in this case to block progress, so I'm fine with the _"if it's defined in Fabricator, it's namespaced"_ methodology if that makes it easier.

> The main reason is that Fabricator's JS toggles both states and utility classes. So it feels weird to me to toggle .f-u-hidden one place but .is-active in another. 

That does seem strange. IMHO it would be better to _only_ use state classes for dynamic JS manipulation. This restriction forces you to think of states more in terms of their behavior than their appearance. 

```css
.f-SomeWidget { 
  /* default state */
}

.f-SomeWidget.is-active {
  /* active state */
}

.f-SomeWidget.is-inactive {
  /* inactive state...could mean that it's hidden */
}
```

But again, this doesn't seem like an overly important matter at the moment, so carry on.